scripts/build-rkt.sh

```diff
@@ -2,4 +2,11 @@
 
 set -e
 
-./autogen.sh && ./configure --enable-tpm=no --with-stage1-default-images-directory=/usr/lib/rkt/stage1-images --with-stage1-default-location=/usr/lib/rkt/stage1-images/stage1-coreos.aci && make -j4
+./autogen.sh && \
```

Comment:
Hm, we have the `set -e` call above so chaining everything with `&&` is not really necessary. Could you simplify it by removing those `&&` and removing one level of indentation in the lines below?